src/screens/CreatePost/index.js

```diff
@@ -135,7 +135,7 @@ const CreatePost = () => {
     hour: 0,
     minute: 0
   });
-  const [expiredSelect, setExpiredSelect] = React.useState(2);
+  const [expiredSelect, setExpiredSelect] = React.useState(params.isCreateCommunity ? 3 : 2);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The initial state of `expiredSelect` is now conditionally set based on the `params.isCreateCommunity`. This change seems to be logical, but it's important to ensure that `params.isCreateCommunity` is always defined and is a boolean to avoid unexpected behavior.
```diff
- const [expiredSelect, setExpiredSelect] = React.useState(params.isCreateCommunity ? 3 : 2);
+ const [expiredSelect, setExpiredSelect] = React.useState(params?.isCreateCommunity === true ? 3 : 2);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->